en/developer_guide/install.rst

```diff
@@ -72,6 +72,12 @@ Install open-orchestra with composer:
     $ ./composer.phar create-project open-orchestra/open-orchestra path/to/your/folder -s dev
     $ ./composer.phar create-project open-orchestra/open-orchestra-front-demo path/to/your/folder -s dev
 
+Clone the provisioning repository :
+
+.. code-block:: bash
+
+    $ git@github.com:open-orchestra/open-orchestra-provision.git
```

Comment:
May be add the entire command 

"git clone git@github.com:open-orchestra/open-orchestra-provision.git"